### PR TITLE
Final mobile portfolio project

### PR DESCRIPTION
**Pull request by Nomarg**
My first portfolio mobile version.
• Added HTML codes
• Added CSS Codes
• Used flex command

### CODE REVIEW

**Review by frank1738 — COMMENTED** [review]
Hi @Nomarg , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted your project on the wrong milestone. You are currently working on  ` Portfolio: mobile set up` but submitted it on ` Portfolio: finish mobile version` . Kindly submit it on the correct milestone

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by frank1738 — COMMENTED** [review]
Hi @Nomarg , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] There seems to be a conflict between your branches that is stopping linter checks from running. Kindly fix that for the linter checks to run

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Eternalgratis — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
Hi @Nomarg , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there

## Highlights
✅ You used the correct GitHub flow
✅ Descriptive pull request message
✅ Linter error checks validated


## Required Changes ♻️
_Check the comments under the review._
- The container of this section of your project should be a section tag. Kindly make the adjustments
- Please put all your project cards in a container and display grid so it can have the column layout as seen in Figma while each of the project cards maintains flexbox just have you have done already. 
- currently, the width of the "see project" button is quite different from that of the Figma. Kindly consider fixing this.
- The "About me section" is a section tag. Please fix this so your code follows the HTML semantics
- Please consider using an hx tag for "Let's connect" because it's a subtitle and not a link based on the Figma design.
- The technology categories "Language, Framework, and skills " is a ul tag while each of them are li
- Each of the technologies "Javascript, HTML , CSS" are ul and li tags


 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -1,180 +1,180 @@
...
+        <h1>Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I’d love to help with it! </p>
+        <input type="text" id="fname" name="fname" value="Lorem Ipsum">
+        <input type="text" id="email" name="email" value="Loremipsum@gmail.com">
+        <input type="text" id="msg" name="msg" value="Write your message here...">
+        <div class="submit">
+            <a href="#" class="tt">Get in touch</a>
+        </div>
+    </div> -->
```
Comment (joskalenda) [inline index.html:180]: - [ OPTIONAL ] It is not recommended to leave commented code in your app as it makes the code base difficult to read, messy, and unprofessional, always take the habit of removing them anytime

File: style.css [hunk 2]
```diff
@@ -1,705 +1,705 @@
...
+  height: 20px;
+}
+
+.social li {
+  padding-right: 24px;
+}
+
+/* project */
+
+.box1 {
```
Comment (joskalenda) [inline style.css]: - [ ] As said by the previous reviewer, you should wrap  all your project cards in a common container and use display:grip property to align its content(projects)

Hint 

```
 <section display:grip>
    `box1 display:flex`
    `box1`
<section/>

```

File: index.html [hunk 3]
```diff
@@ -1,180 +1,180 @@
...
+
+    <section class="hero">
+        <h1>I'm Abhishek. <br> Glad to see <br> you !</h1>
+        <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me. </p>
+        <a href="#">LET'S CONNECT</a>
+            <ul class="social">
+                <li><img src="/image/Vector.svg"></li>
+                <li><img src="/image/Linkedin_icon.svg"></li>
+                <li><img src="/image/Vector1.svg"></li>
+                <li><img src="/image/Vector2.svg"></li>
```
Comment (joskalenda) [inline index.html]: - [ OPTIONAL ] Your images are not showing, I have modified the HTML to add a `.` before the /, and there we go, I think you need to review this part to implement the changes, see?

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @Nomarg , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ]... ✅ Clean design
- [ ].. ✅ Gread HTML code
- [ ].... ✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (shahadat3669) [inline index.html:1]: - [ ] [Optional] Well done! 👍 I can see that you have implemented all the requirements mentioned in the previous review. However, it seems that the responsiveness of your code needs further improvement. When reviewing the design, I noticed that it doesn't look good on different screen sizes. It is important to ensure that your website is responsive and provides a consistent experience across various devices.

As per the design specifications, your website should be mobile-responsive, with a minimum screen size of 375px, and desktop-responsive, with a minimum screen size of 768px. I recommend carefully reviewing the design and making the necessary adjustments to ensure that your code is responsive and displays correctly on different devices.

Keep up the good work, and don't forget to thoroughly test your website on different screen sizes to ensure a seamless user experience!


[My-Portfolio.webm](https://github.com/Nomarg/My-Portfolio/assets/55840999/028ffe6f-2f7c-4489-b331-6f5658a12da5)

**Review by shahadat3669 — APPROVED** [review]
Hi @Nomarg , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Well-structured files ✅
All linter checks are passing ✅

## Optional suggestions
- [Optional] the branch name you have chosen does not provide a clear indication of the feature or action being implemented. It is important to use descriptive branch names that accurately describe the purpose of the branch. For example, instead of a generic name like "feature1," consider using names like "**create_sidebar**" or "**add_comments_section**" that convey the specific feature or action being worked on. :pray: Please rename the branch to a meaningful name. Check out this [official document](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) for further help. :+1:


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**